window-copy.c

```diff
@@ -72,15 +74,20 @@ static int	window_copy_search_marks(struct window_mode_entry *,
 		    struct screen *, int, int);
 static void	window_copy_clear_marks(struct window_mode_entry *);
 static void	window_copy_move_left(struct screen *, u_int *, u_int *, int);
+static void	window_copy_move_right(struct screen *, u_int *, u_int *, int);
+static void	window_copy_move_before_search_mark(
```

Comment:
I wasn't sure if these were necessary, but it seemed like other ones that weren't strictly necessary were declared, so i followed suit.